src/seed_data/utils/utilFunctions.ts

```diff
@@ -0,0 +1,51 @@
+import * as faker from 'faker';
+import { LetterGrade } from '../../entity/Enums';
+
+export function getRandomIntInclusive(min: number, max: number) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1) + min); //The maximum is inclusive and the minimum is inclusive
+}
+
+export function toUpperCase(value: String) {
+    const words = value.split(' ');
+    var i: number;
+    for (i = 0; i < words.length; i++) {
+        words[i] = words[i][0].toUpperCase() + words[i].substr(1);
+    }
+
+    return words.join(' ');
+}
+
+export function chooseFrom(options: any[]) {
+    const x = faker.random.number(options.length - 1);
```

Comment:
i stumbled across this neato function from `faker` that selects items randomly given an array of elements
`faker.random.arrayElement(someArrayHere)`